README.md

```diff
@@ -1,2 +1,223 @@
-# Microverse-Capstone-M1
-It is A Capstone Project in HTML, CSS, and JavaScript Assigned by Microverse in Outcome Based Assessment.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="wahab_icon.jpeg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Capstone Project Module 1</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse Capstone Project Module 1 <a name="about-project"></a>
+
+This is My Capstone Project of Module 1 in Microverse which is A Dynamic and Responsive Website of A Non Profit Organization to Raise Awareness and Funds for The Flood Victims of 2022 in Pakistan.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 2- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile Responsive**
+- **Friendly UX**
+- **Easy to use**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] As mentioned in the previous review, you should deploy the peoject using `github pages` and add the deployed poject link in the `README` file.

Reply:
done.